src/datascience-ui/data-explorer/sliceControl.css

```diff
@@ -125,3 +121,9 @@ details[open] > summary::before {
     border: var(--vscode-dropdown-border);
     color: var(--vscode-dropdown-foreground);
 }
+
+[class*='fieldGroup-77'],
+[class*='fieldGroup-42'] {
```

Comment:
This is a hack for the FluentUI dropdowns. Going to file an issue on them because I don't seem to be able to override all of their styles using their dropdown component's styles prop